docs/api/app.md

```diff
@@ -641,9 +643,9 @@ created by this method. On failure an `Error` is thrown.
 
 You can only override paths of a `name` defined in `app.getPath`.
 
-By default, web pages' cookies and caches will be stored under the `userData`
+By default, web pages' cookies and caches will be stored under the `userCache`
```

Comment:
Is this actually true? Does overriding `DIR_USER_CACHE` in the AtomBrowserContext constructor actually change behavior s.t. all the cache data goes there?

Reply:
This notion of DIR_CACHE, DIR_USER_CACHE already exist in the code but are R/O info, changing them have currently no effect.